crates/orderbook/src/main.rs

```diff
@@ -38,7 +38,7 @@ use shared::{
     },
     baseline_solver::BaseTokens,
     current_block::current_block_stream,
-    http_solver::{DefaultHttpSolverApi, SolverConfig},
+    http_solver::{DefaultHttpSolverApi, Objective, SolverConfig},
```

Comment:
Is this change needed here, or is the new parameter part of `SolverConfig`?

Reply:
Its a new parameter part of `SOlverConfig`.

Driver creates one with default objective (so nothing specified in the query URL) for solving batches and the orderbook creates on with objective set to `surplusfeescosts` for price estimates.